spec/unit/x509/cert_provider_spec.rb

```diff
@@ -307,7 +307,7 @@ def expects_private_file(path)
           # password is 74695716c8b6
           expect {
             provider.load_private_key('encrypted-ec-key')
-          }.to raise_error(OpenSSL::PKey::PKeyError, /Could not parse PKey: no start line/)
+          }.to raise_error(OpenSSL::PKey::PKeyError, /(unknown|invalid) curve name|Could not parse PKey: no start line/)
```

Comment:
Is this different behavior in JRuby vs MRI?

Reply:
yeah exactly